ios/FluentUI.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,17 +6,17 @@
         "repositoryURL": "https://github.com/microsoft/appcenter-sdk-apple.git",
         "state": {
           "branch": null,
-          "revision": "25f64229373de97ff3920941cd52203193e5d8be",
-          "version": "4.3.0"
+          "revision": "8354a50fe01a7e54e196d3b5493b5ab53dd5866a",
+          "version": "4.4.2"
```

Comment:
Were these changes intentional?

Reply:
They were not. What is the general policy for automatic changes like this?

Reply:
I think it should probably be removed from this PR, and if we think we actually need the change it can be added with a separate PR.